vcpkg/commands/add-version.md

```diff
@@ -0,0 +1,52 @@
+---
+title: vcpkg x-add-version
+description: Command line reference for the vcpkg x-add-version command. Update version database of vcpkg ports.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/14/2023
+ms.prod: vcpkg
+---
+
+# vcpkg x-add-version
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+## Synopsis
+
+```console
+vcpkg x-add-version [port-name] [options] [--all] [--overwrite-version] [--skip-formatting-check] [--skip-version-format-check] [--verbose]
+```
+
+## Description
+
+The `x-add-version` command updates the version database for vcpkg ports. By default, it operates on a specified port. With the appropriate switches, users can choose to process all ports or change the default behavior regarding formatting checks and version updates.
+
+## Options
+
+All vcpkg commands support a set of [common options](common-options.md).
+
+### `[port-name]`
+
+Specifies the name of the port to be updated. If not provided, the user should use the `--all` option to process all ports.
+
+### `--all`
+
+Processes all the ports in the built-in ports directory. If this option is used in conjunction with a specified port name, a warning will be issued, and the specific port will be ignored. 
```

Comment:
```suggestion
Processes all the ports in the built-in `ports` directory. If this option is used together with a port name, a warning is issued and the specific port is ignored. 
```

> built-in `ports` directory 

"built-in ports" is a concept so it should have a link to somewhere else in the documentation explaining what it means.

> a warning is issued, and the specific port is ignored. 

Technically, it is not ignored since the provided port name is processed as part of "all" ports.
@AugP thinks this should be a hard error instead of a warning. Not blocking on this since it is documenting existing behavior.